src/commands/image/imageSource/buildImageInAzure/UploadSourceCodeStep.ts

```diff
@@ -13,22 +14,23 @@ import { createActivityChildContext } from '../../../../utils/activity/activityU
 import { createContainerRegistryManagementClient } from '../../../../utils/azureClients';
 import { localize } from '../../../../utils/localize';
 import { BuildImageInAzureImageSourceContext } from './BuildImageInAzureContext';
-import path = require('path');
 
 const vcsIgnoreList = ['.git', '.gitignore', '.bzr', 'bzrignore', '.hg', '.hgignore', '.svn'];
 
 export class UploadSourceCodeStep extends ExecuteActivityOutputStepBase<BuildImageInAzureImageSourceContext> {
     public priority: number = 430;
+    private _sourceFilePath: string;
 
     protected async executeCore(context: BuildImageInAzureImageSourceContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
         context.registryName = nonNullValue(context.registry?.name);
         context.resourceGroupName = getResourceGroupFromId(nonNullValue(context.registry?.id));
         context.client = await createContainerRegistryManagementClient(context);
+        /* relative path of src folder from rootFolder and what gets deployed*/
+        this._sourceFilePath = path.dirname(path.relative(context.rootFolder.uri.path, context.dockerfilePath));
 
-        const uploading: string = localize('uploadingSourceCode', 'Uploading source code...');
+        const uploading: string = localize('uploadingSourceCode', 'Uploading source code from "{0}" directory...', this._sourceFilePath);
```

Comment:
My vote is to keep the progress report message the old way, it's a little easier on the eyes in my opinion and we can see the longer versions in the output and activity log later.

Reply:
kk that's fine with me